src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationIncorrect.java

```diff
@@ -11,76 +11,92 @@
 
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
 
-@MyAnn_21 @com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation.MyAnn // violation
+// violation below 'Annotation 'MyAnn' should be alone on line.'
+@MyAnn_21 @com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation.MyAnn
 (value = "")
 class InputAnnotationLocationIncorrect
 {
 
-    @MyAnn_21 @MyAnnotation1(value = "") // violation
+    @MyAnn_21 @MyAnnotation1(value = "") // violation '.*'MyAnnotation1' should be alone on line.'
     public int a;
 
-    @MyAnnotation1(value = "") public int b; // violation
+    // violation below 'Annotation 'MyAnnotation1' should be alone on line.'
+    @MyAnnotation1(value = "") public int b;
 
     @MyAnn_21 // ok
-        @MyAnnotation1 // violation
+        // violation below '.* 'MyAnnotation1' have incorrect indentation level 8, .* should be 4.'
+        @MyAnnotation1
```

Comment:
Please keep trailing by comment when it is possible.
`violation below '.* incorrect .* level 8, .* should be 4.'` is most compact version that still keep meaning.